05week/spaceTravelToMars.js

```diff
@@ -10,19 +10,54 @@ let jobTypes = {
 };
 
 // Your code here
+class CrewMember {
+  constructor(name, job, specialSkill, ship) {
+    //Set all the initial values
+    this.name = name;
+    this.job = job;
+    this.specialSkill = specialSkill;
+    this.ship = ship;
+  }
+  //enterShip = (ship) =>{
+  enterShip(ship){
+    //Set the ship data
+    this.ship = ship;
+    ship.crew[0] = this;
```

Comment:
this would only let the ship have one crew member, it's an array, it's supposed to allow for as many crew members that enter the ship. What is another way that you could put the crew member on to the ship's crew?